### PR TITLE
Stress 2

### PR DESCRIPTION
**Pull request by bplantico**
What does this PR do?

This pull request adds the previously missing destroy action to the admin tutorial controller so that an admin has the ability to delete a tutorial.

### CODE REVIEW

File: app/javascript/packs/application.js [hunk 1]
```diff
@@ -4,3 +4,6 @@ import { definitionsFromContext } from "stimulus/webpack-helpers"
 const application = Application.start()
 const context = require.context("./controllers", true, /\.js$/)
 application.load(definitionsFromContext(context))
+
+import Rails from 'rails-ujs';
```
[the remark this answers is not in the document]
Comment (bplantico) [reply app/javascript/packs/application.js:8]: This works as written so ignoring Hound suggestion.

File: db/seeds.rb [hunk 2]
```diff
@@ -125,7 +125,18 @@
   'position' => 6
 })
 
+mod_3fe_tutorial_data = {
+  'title' => 'Front End Engineering - Module 3',
+  'description' => 'Videos related to Front End Mod 3.',
+  'thumbnail' => 'https://i.ytimg.com/vi/tZDBWXZzLPk/hqdefault.jpg',
+  'classroom' => true
```
Comment (alexander-mathieu) [inline db/seeds.rb:132]: This will be good for the demo.

File: db/seeds.rb [hunk 3]
```diff
@@ -125,7 +125,18 @@
...
+}
+
+mod_3fe_tutorial = Tutorial.create! mod_3fe_tutorial_data
+
 User.create!(email: 'admin@example.com', first_name: 'Bossy', last_name: 'McBosserton', password: 'password', role: :admin, verified_email: true)
 User.create!(email: 'user1@example.com', first_name: 'Ricky', last_name: 'Bobby', password: 'password', github_token: ENV['GITHUB-TOKEN'], github_login: 'number1orbust', role: :default, verified_email: true)
 User.create!(email: 'user2@example.com', first_name: 'Jean', last_name: 'Girard', password: 'password', verified_email: true)
 User.create!(email: 'user3@example.com', first_name: 'Cal', last_name: 'Naughton, Jr.', password: 'password', verified_email: true)
+User.create!(email: 'corndog@example.com', first_name: 'Kyle', last_name: 'Corndog', password: 'password', verified_email: true, github_uid: '46171611')
```
Comment (alexander-mathieu) [inline db/seeds.rb:141]: 🌽 🐶

File: README.md [hunk 4]
```diff
@@ -47,6 +47,9 @@ $ rake db:create
 $ rake db:migrate
 $ rake db:seed
 ```
+IF VIDEOS NOT LOADING DUE TO MISSING POSITIONS:
+Run from command line:
```
Comment (alexander-mathieu) [inline README.md:51]: Nice README work!

File: app/controllers/admin/tutorials_controller.rb [hunk 5]
```diff
@@ -18,6 +18,13 @@ def update
     redirect_to edit_admin_tutorial_path(tutorial)
   end
 
+  def destroy
+    tutorial = Tutorial.find(params[:id])
+    tutorial.destroy
```
Comment (alexander-mathieu) [inline app/controllers/admin/tutorials_controller.rb:23]: It's a wonder this was even missing in the first place.

File: app/javascript/packs/application.js (hunk 1 above)
Comment (alexander-mathieu) [inline app/javascript/packs/application.js:8]: 👍

File: app/views/admin/dashboard/show.html.erb [hunk 6]
```diff
@@ -15,7 +15,7 @@
       <td><%= link_to tutorial.title, tutorial_path(tutorial) %></td>
       <td>
         <%= link_to "Edit", edit_admin_tutorial_path(tutorial) %> |
-        <%= link_to 'Destroy', tutorial_path(tutorial),
+        <%= link_to 'Destroy', admin_delete_tutorial_path(tutorial),
```
Comment (alexander-mathieu) [inline app/views/admin/dashboard/show.html.erb:18]: 👍

**Review by alexander-mathieu — APPROVED** [review]
Looks good.

**Pull request merged**